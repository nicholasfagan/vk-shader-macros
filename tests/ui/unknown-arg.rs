use vk_shader_macros::include_glsl;

const VERTEX: &[u32] = include_glsl!(r##"src"##, bad_arg: arg);

fn main() {}