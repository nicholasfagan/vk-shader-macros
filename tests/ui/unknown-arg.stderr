error: unknown shader compile argument
See `https://docs.rs/vk-shader-macros/0.2.6' for help
 --> $DIR/unknown-arg.rs:3:50
  |
3 | const VERTEX: &[u32] = include_glsl!(r##"src"##, bad_arg: arg);
  |                                                  ^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
